src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Avatar from "../Avatar/Avatar";
+import Counter from "../Counter/Counter";
+import Tooltip from "../Tooltip/Tooltip";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import styles from "./AvatarGroup.module.scss";
+
+const AvatarGroup = ({
+  className,
+  id,
+  children,
+  size,
+  type,
+  max,
+  counterProps,
+  counterTooltipCustomProps,
+  counterTooltipIsVirtualizedList,
```

Comment:
right now we don't support it no? so can be removed. also when we will support it we need to find more simple name

Reply:
Are you talking about `counterTooltipIsVirtualizedList`? We do support this, but only for auto-generated tooltip. I've added example in the storybook
We can discuss a better name